config-options/xml-configuration/README.adoc

```diff
@@ -1,30 +1,48 @@
 = XML Configuration
 
 This example demonstrates how to use XML configuration instead of the java config-api,
- to configure WildFly Swarm.
+to configure WildFly Swarm. This might be useful, when you are migrating from a traditional
+WildFly deployment and you have your `standalone.xml` file ready.
 
 
 == Referencing standalone.xml
 
-
 To reference a `standalone.xml` it can be part of your project, where it's picked up automatically,
 or you can pass it to WildFly Swarm with a `-c` parameter:
 
-[source,xml]
+[source,sh]
 ----
-include::pom.xml[lines=39..45]
+cd xml-configuration
+java -jar target/example-xml-config-swarm.jar -c ../standalone.xml
 ----
 
-See: link:pom.xml#L42[pom.xml]
-
 == Subsystem configuration in XML
 
 You can then have any subsystem configuration external in `standalone.xml`.
 These configuration will then be matched with the corresponding fractions automatically.
 
+In in our example link:../standalone.xml#L130-L147[standalone.xml] file, we configure a custom datasource
+called `CustomDS`:
+
 [source,xml]
 ----
-include::../standalone.xml[lines=130..147]
+<subsystem xmlns="urn:jboss:domain:datasources:4.0">
+  <datasources>
+    <datasource jndi-name="java:jboss/datasources/CustomDS" pool-name="BSDS" enabled="true" use-java-context="true">
+      <connection-url>jdbc:h2:mem:test;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE</connection-url>
+      <driver>h2</driver>
+      <security>
+        <user-name>sa</user-name>
+        <password>sa</password>
+      </security>
+    </datasource>
+    ...
+</subsystem>
 ----
 
-See: link:../standalone.xml#L130[standalone.xml]
+To demonstrate that the configuration woks, we use the `CustomDS` datasource in a JAX-RS endpoint
```

Comment:
> woks -> works

Fixed in https://github.com/wildfly-swarm/wildfly-swarm-examples/pull/136/commits/f305e2ef848bfa35de6eb70708deb4e0eb958a0c#diff-27cc72a505e4679a493a9d7b8a3e91daR43